client/src/components/auth/Login.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from "react";
+import Footer from "../Footer.js";
+import { loginFail } from "../../store/actions/userActions";
+import { connect } from "react-redux";
+import axios from "axios";
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: ""
+    };
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.errorMessage = this.errorMessage.bind(this);
+  }
+
+  onSubmit(e) {
+    e.preventDefault();
+    axios
```

Comment:
For best practice, you should keep side effects in action thunks.
https://medium.com/fullstack-academy/thunks-in-redux-the-basics-85e538a3fe60